### PR TITLE
bsunanda:Run2-hcx82 Add the possibility of a FrontEnd map for HCAL

### PR DESCRIPTION
**Pull request by bsunanda**
This will enable the use of FrontEnd elements in categorizing noise

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@ggovi @mmusich @franzoni We would like to have this integrated so that we can make a CMSSW version working for additional channels (HF and HE)

**Comment by abdoulline** [comment]
@ggovi @mmusich @franzoni  - please consider revising+aprroving it

**Comment by abdoulline** [comment]
@ggovi  @mmusich  @franzoni  
-  Gentlemen?

File: CondTools/Hcal/src/HcalFrontEndMapHandler.cc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+HcalFrontEndMapHandler::~HcalFrontEndMapHandler()
+{
+}
+
+void HcalFrontEndMapHandler::getNewObjects()
+{
+  //  edm::LogInfo   ("HcalFrontEndMapHandler") 
+  std::cout
+    << "------- " << m_name
```
Comment (mmusich) [inline CondTools/Hcal/src/HcalFrontEndMapHandler.cc]: @bsunanda please use `LogInfo` instead of `cout`

**Comment by mmusich** [comment]
+1
@bsunanda excluding minor comment I think this is fine. I let @ggovi comment for the new `CondFormat`:  `CondFormats/HcalObjects/src/HcalFrontEndMap.cc`

**Comment by bsunanda** [comment]
@ggovi Please approve this. We need to populate the DB with front end map urgently

**Comment by abdoulline** [comment]
@ggovi -  Giacomo,  HCAL needs it in 81X asap, it cannot just stay idle/unattended...

**Comment by ggovi** [comment]
+1

**Pull request merged**